demos/custom-routes/test/index.test.js

```diff
@@ -0,0 +1,2543 @@
+/* eslint-env jest */
```

Comment:
Do these tests pass? Do we want to run them in CI?

Reply:
I didn;t mean to add them. I'd not committed them originally, but then accidentally added them when rebasing

Reply:
(They need rewriting to work with Netlify)